cypress/integration/multi_select_spec.js

```diff
@@ -0,0 +1,142 @@
+context('Single Select', () => {
+    beforeEach(() => {
+      cy.visit('./examples/MultiSelect/index.html')
+    })
+  
+    // describe('disabled', () => {
+    //   it('cannot be opened by clicking on it', () => {
+    //     // disable select first
+    //   })
+    // })
+  
+    describe('enabled', () => {
```

Comment:
Remove unnecessary code comments and provide a better description for the 'describe'